test/navigation/AuthorisedOfficialsNavigatorSpec.scala

```diff
@@ -204,6 +205,14 @@ class AuthorisedOfficialsNavigatorSpec extends SpecBase {
                 .success.value) mustBe
               authOfficialRoutes.IsAuthorisedOfficialPreviousAddressController.onPageLoad(NormalMode, index)
           }
+
+          "go to the Amend address page if one or more address lines length >35 characters when clicked Confirm and continue button" in {
```

Comment:
can we get a test for special characters too

Reply:
All amend address pages using same html and form provider, so I wrote test for special characters in 
[https://github.com/hmrc/charities-registration-frontend/pull/439/files#diff-29efb6569bd2837394c1f62964d46176d16940d8ded36782d8a8e58ca8ebcac1R140]